pydantic_ai_slim/pydantic_ai/fastapi/registry.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from functools import lru_cache
+
+from pydantic_ai import Agent
+
+logger = logging.getLogger(__name__)
+
+
+class AgentRegistry:
```

Comment:
I don't think we should work on a registry basis.

I think the `Agent` object should expose an ASGI application, that you can expose.

If you need multiple agents to become apps/endpoints, you can mount them together.